packages/package_bioformats_1_0_4/tool_dependencies.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0"?>
+<tool_dependency>
+    <package name="javabridge" version="1.0.11">
+        <repository name="package_javabridge_1_0_11" owner="iuc" prior_installation_required="True" />
+    </package>
+    <package name="bioformats" version="1.0.4">
+        <install version="1.0">
+            <actions>
+                <action type="download_by_url">https://pypi.python.org/packages/source/p/python-bioformats/python-bioformats-1.0.4.tar.gz#md5=b5b316f9da734d82ebb0254f22c30b1f</action>
+                <action type="make_directory">$INSTALL_DIR/lib/python</action>
+                <action type="set_environment_for_install">
+                    <repository name="package_atlas_3_10" owner="iuc">
+                        <package name="atlas" version="3.10.2" />
+                    </repository>
+                    <repository name="package_numpy_1_9" owner="iuc">
+                        <package name="numpy" version="1.9" />
+                    </repository>
+                    <repository name="package_javabridge_1_0_11" owner="iuc">
+                        <package name="javabridge" version="1.0.11" />
+                    </repository>
+                </action>
+                <action type="shell_command">
+                    export PYTHONPATH=$PYTHONPATH:$INSTALL_DIR/lib/python &amp;&amp; 
+                    python setup.py install --install-lib $INSTALL_DIR/lib/python
+                </action>
+                <action type="set_environment">
+                    <environment_variable action="prepend_to" name="PYTHONPATH">$INSTALL_DIR/lib/python</environment_variable>
```

Comment:
Sure!  I'll also rename to python_bioformats if you agree.  Probably a better name since it downloads the python-bioformats module.  https://pypi.python.org/pypi/python-bioformats  Let me know if you'd rather name it java_bioformats as you stated below.